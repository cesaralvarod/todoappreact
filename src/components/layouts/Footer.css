footer {
  background: #eee;
  padding: 2rem;
  margin-top: 3rem;
}

footer a,
footer a:visited {
  text-decoration: none;
  color: #d4983f;
}
