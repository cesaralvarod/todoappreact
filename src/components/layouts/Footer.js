import React from "react";

// Styles

import "./Footer.css";

const Footer = () => {
  return (
    <footer>
      <p>
        &copy; <a href="https://github.com/cesaralvarod">cesaralvarod</a> 2021
      </p>
    </footer>
  );
};

export default Footer;
