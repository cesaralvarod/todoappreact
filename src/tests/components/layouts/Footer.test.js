import React from "react";
import { shallow } from "enzyme";
import Footer from "../../../components/layouts/Footer";

describe("Testing in component layout Footer", () => {
  test("Component layout Footer should show correctly", () => {
    const wrapper = shallow(<Footer />);

    expect(wrapper).toMatchSnapshot();
  });
});
