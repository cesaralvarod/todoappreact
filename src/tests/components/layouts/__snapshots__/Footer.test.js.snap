// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing in component layout Footer Component layout Footer should show correctly 1`] = `
<footer>
  <p>
    Â© 
    <a
      href="https://github.com/cesaralvarod"
    >
      cesaralvarod
    </a>
     2021
  </p>
</footer>
`;
